### PR TITLE
Refresh packages to latest available versions in tools image

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
VRs : all jobs from 
http://autobot.qe.nue2.suse.org/tests/4165
till
http://autobot.qe.nue2.suse.org/tests/4183

**Review by mpagot — APPROVED** [review]
LGTM

File: tests/publiccloud/prepare_tools.pm [hunk 1]
```diff
@@ -71,14 +74,24 @@ sub run {
...
 
     # Install prerequisite packages test
-    zypper_call('-q in python310-pip python310-devel python3-pytest python3-img-proof python3-img-proof-tests podman docker jq rsync unzip');
-    record_info('python', script_output('python --version'));
+    zypper_call('-q in python3-img-proof python3-img-proof-tests');
+    record_info('python exec', script_output("$python_exec --version"));
+
+    # bsc#1213529
+    assert_script_run('cat /usr/lib/python3.6/site-packages/azure_core-1.23.1-py3.6.egg-info/requires.txt');
+    assert_script_run(q(sed -i 's/^typing-extensions>=4\.0\.1$/typing-extensions>=3.10.0.0/' /usr/lib/python3.6/site-packages/azure_core-1.23.1-py3.6.egg-info/requires.txt));
```
Comment (pdostal) [inline tests/publiccloud/prepare_tools.pm:82]: Is this something we can maybe test in our CLI test too?

**Review by pdostal — APPROVED** [review]
Nice :)

**Review by ricardobranco777 — APPROVED** [review]

**Review by mimi1vx — APPROVED** [review]

**Pull request merged**